.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: node_js
+node_js:
+  - "stable"
+services:
+  - postgresql
+notifications:
+  email: false
```

Comment:
This requires a new line to be present at the end of the file

Reply:
ok @eguonoefe